Makefile

```diff
@@ -1,13 +1,19 @@
-build:
+.PHONY: build clean test
+
+build: clean
 	git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-	pip install ./sdx-common
-	pip3 install -r requirements.txt
+	pip3 install ./sdx-common
 	rm -rf sdx-common
+	pip3 install -r requirements.txt
+
+dev:
+	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
```

Comment:
The `dev` target should be removed altogether - should be the same process for dev and prod deployment